clang/lib/Parse/ParseDecl.cpp

```diff
@@ -7986,6 +7986,12 @@ void Parser::ParseParameterDeclarationClause(
     if (getLangOpts().HLSL)
       MaybeParseHLSLAnnotations(DS.getAttributes());
 
+    if (ParmDeclarator.getDeclarationAttributes().size() &&
```

Comment:
```suggestion
    if (!ParmDeclarator.getDeclarationAttributes().empty() &&
```

Reply:
@erichkeane thanks. changed to use `empty`